docs/takeaway.rb

```diff
@@ -0,0 +1,60 @@
+class Takeaway
+
+attr_reader :table_order
+
+  def initialize
+    @table_order = Array.new
```

Comment:
Style/EmptyLiteral: Use array literal `[]` instead of `Array.new`. (https://github.com/bbatsov/ruby-style-guide#literal-array-hash)